test/mocha/20-recovery.js

```diff
@@ -26,6 +26,7 @@ describe('Failure recovery', function() {
     agents = await helpers.insertIssuerAgent({
       id: accountId,
       token: 'token-43f47a1f-acaf-4dd1-8597-001a8b0637e3',
+      didMethod: 'key',
```

Comment:
can we not recover using `didMethod: 'v1'`?

Reply:
added a test here 699aa4b